### PR TITLE
Updated url to always point to the matching versioned doc.

### PR DESCRIPTION
**Pull request by adrianjgeorge**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -101,6 +100,10 @@ allprojects {
             url "https://maven.research.rackspacecloud.com/content/groups/aggregate/"
         }
     }
+
+    ext {
+        docsHome = "http://www.openrepose.org/versions/$version/index.html".toString()
+    }
```
Comment (dmnjohns) [inline build.gradle]: What is the difference between putting the external property on every project versus putting it on the root project? Out of curiosity.

Comment (dmnjohns) [inline build.gradle]: Oh, I see. Was this just done to get the version in scope? Does the version not get inherited from the parent? Could that possible be extracted out of the `allprojects` block?

Comment (adrianjgeorge) [inline build.gradle]: I originally had it above, but version is not yet defined at that point. And for other reasons version can't by in the outer scope, so it has to be down here.

Comment (dmnjohns) [inline build.gradle]: Fair enough.

**Pull request merged**